pkg/hubble/metrics/api/context.go

```diff
@@ -151,6 +187,20 @@ func sourcePodShortContext(flow *pb.Flow) (context string) {
 	return
 }
 
+func sourceDNSContext(flow *pb.Flow) (context string) {
+	if flow.GetSourceNames() != nil {
+		context = strings.Join(flow.GetSourceNames(), ",")
```

Comment:
do you need to sort the dns name list, or is it already guaranteed to be sorted?

Reply:
Thanks for reminding me to check this, they are guaranteed to be sorted: https://github.com/cilium/cilium/blob/87b56a42a56633f1448c0c99a75dc26e5bb4c29f/pkg/fqdn/cache.go#L467